osi_hostvehicledata.proto

```diff
@@ -2,31 +2,354 @@ syntax = "proto2";
+            // wheel would be (axle=1, index=0).
+            //
+            optional uint32 index = 2;
+
+            // Dry friction is a force that opposes the relative lateral motion of two solid surfaces
+            // in contact. It is subdivided into static friction between non-moving surfaces and kinetic
+            // friction between moving surfaces.
+            // Ued here is the dry friction coefficient of the paired materials (see reference).
+            // Dimensionless.
+            //
+            // Unit: -
+            //
+            // \par References:
+            // - http://adaptivemap.ma.psu.edu/websites/6_friction/dry_friction/dryfriction.html
+            //
+            //optional double kinetic_friction_coefficient = 3;
+
+            // Contact point of the mid of the tire tread with the world.
+            // Absolute coordinates (x, y, z).
+            //
+            // Unit: m
+            //
+            //optional Vector3d contact_point = 4;
+
+            // Contains the rotation rate of the wheel per second in radians.
+            // Based on the processed output of the hall sensor at the wheel.
+            //
+            // Unit: rad/s
+            //
+            optional double rotation_rate = 5;
+
+            // Contains the longitudinal slip of the tire.
+            // \par References:
+            // - https://www.kfz-tech.de/Biblio/Formelsammlung/Schlupf.htm
+            //
+            // Unit: %
+            //
+            optional double slip = 6;
+        }
+    }
+
+    //
+    // \brief Current calculated and estimated location that can be based on GNSS- and related navigation sensors,
+    // but this message does not contain the single sensor values of the sensorics.
+    //
+    // This message contains the most accurate information the vehicle knows about its positioning and kinematics
+    // available on the board net.
+    // Because of this the values can differ from the "true" values calculated out of
+    // GroundTruth::proj_string, GroundTruth::MovingObject::BaseMoving::position, GroundTruth::host_vehicle_id.
+    //
+    message VehiclePositionAndKinematics
+    {
+        // Geodetic origin of the ENU (east-north-up) cartesian coordinate system regarding WGS84.
+        //
+        optional GeodeticPosition enu_origin = 1;
+
+        // The host vehicle location and kinematics at \c HostVehicleData::timestamp.
+        //
+        // \note If \c HostVehicleData::enu_origin is specified, then the \c BaseMoving parent frame used by
+        // \c HostVehicleData::location is the ENU frame according to ISO8855 with the geodetic origin of
+        // \c HostVehicleData::enu_origin. Otherwise \c BaseMoving parent frame is an arbitrary cartesian coordinate system
+        // with its axis conventions following ISO8855.
+        //
+        // \note Note that dimension and base_polygon need not be set.
+        //
+        optional BaseMoving location = 2;
```

Comment:
@u21n80 : I am still not convinced about BaseMoving with a missing acceleration description..

Reply:
Basically I descripted the parent frame and in [BaseMoving](https://opensimulationinterface.github.io/open-simulation-interface/structosi3_1_1BaseMoving.html#a88af69f98a0768fe9d38189ea5d4fb25) you can see that the description refers to that  parent frame.
```
The relative acceleration of the moving object w.r.t. its parent frame, noted in the parent frame. The acceleration becomes global/absolute if the parent frame is inertial (all parent frames up to ground truth).
```
This should make a clear description.

Reply:
What I meant is the relation to board net signals, where does the acceleration come from, where are they measured in the car? The true acceleration is already in groundtruth e.g.